src/Scheduler.ts

```diff
@@ -6,6 +6,6 @@ export interface Scheduler {
   schedule<T>(work: (state?: T) => Subscription | void, delay?: number, state?: T): Subscription;
   flush(): void;
   active: boolean;
-  actions: Action[];
+  actions: Action<any>[]; // XXX: use `any` to remove type param `T` from `Scheduler`.
```

Comment:
> Should we add T to Scheduler for more strongly typing?

This is a separated issue (Sorry!). [In this PR, I don't discuss about it](https://github.com/ReactiveX/rxjs/pull/1522#issuecomment-202701079)